staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3334,6 +3334,32 @@ const (
 	ScheduleAnyway UnsatisfiableConstraintAction = "ScheduleAnyway"
 )
 
+// PolicyName defines the type of node inclusion policy
+// +enum
```

Comment:
In terms of API enforcement / code generation, does this tag have a real effect?

Reply:
I think they're the main benefits. `enum` marker was introduced by this kep https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2887-openapi-enum-types/README.md, you can also refer to this if you want.

Reply:
Neat

Reply:
Note: Also you can check the generated swagger.json to see the enum tag in action.